backend/src/main/java/gov/cdc/usds/simplereport/api/model/errors/InvalidBulkTestResultUploadException.java

```diff
@@ -0,0 +1,11 @@
+package gov.cdc.usds.simplereport.api.model.errors;
```

Comment:
does this make more sense in the service package since that's where we throw it?

Reply:
Why not! [f42fef4](https://github.com/CDCgov/prime-simplereport/pull/3766/commits/f42fef43666c38a58b222aa3869b22d486727ccb)